src/azure-cli/HISTORY.rst

```diff
@@ -21,6 +21,10 @@ Release History
 * [BREAKING] When not specified, the default value for `--start-task-wait-for-success` on `az batch pool create` is now true (was false).
 * [BREAKING] The default value for Scope on AutoUserSpecification is now always Pool (was Task on Windows nodes, Pool on Linux nodes). This argument is not exposed via the commandline, but can be set in the `--json-file` arguments.
 
+**Compute**
+
+* vmss create/update: Add --terminate-notification and --terminate-notification-time parameters to support terminate scheduled event configurability.
```

Comment:
For `create` command, we probably only need `--terminate-notification-time`. as long as the time argument value is provided, CLI should enable the notification. I assume by default the notification is off, right?

Reply:
Yes. It is turned off by default. OK.